### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kejadlen**

### CODE REVIEW

File: src/stackdriver-nozzle/README.md [hunk 1]
```diff
@@ -1,19 +1,86 @@
+## stackdriver-nozzle
```
Comment (johnsonj) [inline src/stackdriver-nozzle/README.md]: Can we get a little blurb here about what it is

File: src/stackdriver-nozzle/README.md [hunk 2]
```diff
@@ -1,19 +1,86 @@
...
+
+`stackdriver-nozzle` is configured through the following environment variables:
+
+#### Firehose
+
+- `FIREHOSE_ENDPOINT` - the CF API endpoint; e.g., `https://api.bosh-lite.com'
+- `FIREHOSE_EVENTS` - comma-separated list of events to pass to Stackdriver;
+  valid events are `LogMessage`, `ValueMetric`, `CounterEvent`, `Error`,
+  `ContainerMetric`, `HttpStart`, `HttpStop`, and `HttpStartStop`
+- `FIREHOSE_USERNAME` - CF username; defaults to `admin`
```
Comment (johnsonj) [inline src/stackdriver-nozzle/README.md]: should we put some info on permissions required by this user, here or in its own section?

Comment (kejadlen) [inline src/stackdriver-nozzle/README.md]: Yup, that's a good idea... I don't know what they are though. 😅

Comment (cholick) [inline src/stackdriver-nozzle/README.md]: openid might not actually be required.

**Pull request merged**